doc-content/enterprise-only/processes/process-instance-details-con.adoc

```diff
@@ -15,8 +15,8 @@ image::processes/processinstancedetails.png[]
 * The *Instance Details* tab: This gives you a quick overview about what is going on inside the process. It displays the current state of the instance and the current activity that is being executed.
 * The *Process Variables* tab: This displays all of the process variables that are being manipulated by the instance, with the exception of the variables that contain documents. Additionally, you can edit the process variable value and view its history.
 * The *Documents* tab: This displays process documents if the process contains a variable of the type *org.jbpm.Document*. This enables easy access, download, and manipulation of the attached documents.
-* The *Logs* tab: This displays business and technical logs for the respective end users. 
-* The *Diagram* tab: This tracks the progress of the process instance through the BPMN2 diagram. The node or nodes of the process flow that are in progress are highlighted in red. 
+* The *Logs* tab: This displays business and technical logs for the respective end users. For more information, see <<{enterprise-dir}/processes/interacting-with-processes-viewing-process-instance-history-log-proc.adoc#,Viewing process instance logs>>.
```

Comment:
"This displays business and technical logs" -> business vs. technical logs views have been consolidated into one view in 7.2. So there's no more business vs. technical logs distinction.
Probably simplify to "This displays process instance logs"

Reply:
Changed as suggested.